drivers/docker/utils.go

```diff
@@ -13,16 +13,25 @@ import (
 	"runtime"
 	"strings"
 
+	"github.com/distribution/reference"
 	"github.com/docker/cli/cli/config/configfile"
 	"github.com/docker/cli/cli/config/types"
-	"github.com/docker/distribution/reference"
 	registrytypes "github.com/docker/docker/api/types/registry"
 	"github.com/docker/docker/registry"
-	docker "github.com/fsouza/go-dockerclient"
 )
 
 func parseDockerImage(image string) (repo, tag string) {
```

Comment:
what is the expected input to this now? how come `docker.ParseRepositoryTag` doesn't work anymore?

Reply:
`docker.ParseRepositoryTag` comes from fsouza client. I just incorporated the logic into our `parseDockerImage` function.